analytics_data_api/v0/models.py

```diff
@@ -214,6 +216,27 @@ class Meta(BaseVideo.Meta):
 
 
 class RosterEntry(DocType):
+
+    course_id = String()
+    username = String()
+    name = String()
+    email = String()
+    enrollment_mode = String()
+    cohort = String()
+    segments = String(fields={'raw': String()})
```

Comment:
Ah, you're right.  This was an attempt at making it a list that I forgot to update.  I'm going to make it a `String` and add a note that this is a list.